extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -150,13 +148,13 @@ export class AmpAdUIHandler {
    * @private
    */
   displayNoContentUI_() {
-    if (this.baseInstance_.getFallback()) {
+    if (this.fallback_) {
```

Comment:
We could have added a "holder" fallback. I'd say remove the `#fallback` and `#holder` properties entirely.